src/components/submissionPage/SubmissionTextBox.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { useState, useEffect } from 'react';
+import Snap from '../../firebase/snap/Snap';
+import SubmitSnap from '../../firebase/snap/SubmitSnap';
+
+
+const SubmissionTextBox: React.FunctionComponent = () => {
+  /* Containing body of the snap */
+  const [message, setMessage] = useState<string>("");
+  const [confirmation, setConfirmation] = useState<Boolean>(false);
+  const [error, setError] = useState<string>("");
+
+  function handleSubmit(event) {
+    //TODO: Get auth().userID
+    event.preventDefault();
+    const resultingSnap: Snap = {
+      to: [""],
```

Comment:
```suggestion
      to: [],
```
This should just be an empty list right?

Reply:
Yup